test/e2e/utils_test.go

```diff
@@ -1101,6 +1123,28 @@ func updateCRPWithTolerations(tolerations []placementv1beta1.Toleration) {
 	}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement with tolerations %s", crpName)
 }
 
+func buildClusterNamesWithoutTaints(taintClusterNames []string) []string {
```

Comment:
Same issue for this utility as well -> it seems that we could just compose the names directly